ingress/controllers/nginx/Makefile

```diff
@@ -4,8 +4,16 @@ all: push
 TAG = 0.5
 PREFIX = gcr.io/google_containers/nginx-ingress-controller
 
+REPO_INFO=$(shell git config --get remote.origin.url)
```

Comment:
Please update the readme with instructions on how to build since it's now a little more than just running make all. Or write a RELEASES.md file like we have in kube-dns: https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/kube2sky/RELEASES.md